zk-wave draft update 1

ltfschoen opened the pull request.

taxmeifyoucan reviewed (changes requested):
Great to see you jumping in and contributing already @ltfschoen! 

Only thing I have to give you a feedback about - it's very early now to already create a project doc and define your plan. My general recommendation would be to maybe keep it in an external doc for now and wait at least till first weeks of the cohort to discuss it with other fellows and mentors. Of course, feel free to keep it here and update it over time, especially  if you are sure this is the path you are taking. It's totally up to you, my comment is just a suggestion, not a requirement.

taxmeifyoucan commented:
Could you please update to PR so doesn't have conflicts with the latest docs and removes the project doc for now? Then I can finally merge it

ltfschoen commented:
Hey Mário, i've rebased with latest master, moved my draft project notes to hackmd, and provided links to the week 0 and week 1 updates, so i think ready to merge

taxmeifyoucan commented:
Great, thank you Luke!
